earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -509,7 +509,7 @@ ()
     typedef ${name}AssetVersionImplD VerImpl;
     Mutable${name}AssetVersionD newver(std::make_shared<VerImpl>(this));
 
-    AddVersionRef(newver->GetRef().toString());
+    AddVersionRef(std::string(newver->GetRef()));
```

Comment:
Minor: do we need the explicit std::string conversion here (and everywhere else it's used?)

Reply:
To me, it seemed like a better way to do things than `.toString()` i.e. it looks more like c++ than java

Reply:
I think it's fine, I'm just not sure it's necessary. It seems like `AddVersionRef(newver->GetRef());` should work with no explicit conversion.